ssntp/README.md

```diff
@@ -97,6 +97,31 @@ UUIDs.
 3. Connection is successfully established. Both ends of the connection
    can now asynchronously send SSNTP frames.
 
+## CIAO Cert ##
+
+SSNTP uses ciao-cert to generate the certificates it needs to communicate. They
+can be generated as follows.
+
+```
+sudo mkdir -p /etc/pki/ciao/
+sudo mkdir -p /var/lib/ciao/logs/scheduler
+sudo -E $GOPATH/bin/ciao-cert -directory /etc/pki/ciao -host localhost -server -role scheduler
```

Comment:
There are more options to ciao-cert and this is only one example. So I would not include those into the overall SSNTP readme but rather somewhere in the wiki.
Also, there are a few examples here as well: https://github.com/01org/ciao/tree/master/ciao-cert